services/src/main/java/org/exoplatform/officeonline/rest/WOPIResource.java

```diff
@@ -183,9 +183,11 @@ public Response putFile(@Context UriInfo uriInfo,
                           @HeaderParam(OVERRIDE) Operation operation,
                           @PathParam("fileId") String fileId) {
 
+    logDebug("WOPI Request handled: putFile");
```

Comment:
It's not right approach, it will print code line of logDebug() method, not an actual line where you wanted to log something. Logging it's not a place where code reuse via function will work in Java ;)

Reply:
Fixed: https://github.com/exo-addons/office-online/pull/11/commits/69919a5da319a67c234e556b535a4523c7cab06c